libtiff/libtiff_ctypes.py

```diff
@@ -107,7 +109,14 @@
     f = open(include_tiff_h, 'r')
     lst = []
     d = {}
+    mergeContinuedLine = None
     for line in f.readlines():
+        if mergeContinuedLine is not None:
+           line = mergeContinuedLine + line
+           mergeContinuedLine = None
+        if line.rstrip().endswith(' \\'):
+            mergeContinuedLine = line.rstrip()[:-1]
+            continue
```

Comment:
Apparently I didn't setup my notifications correctly and I didn't see this before making #148.

This logic here could be simplified by using a while loop and in my PR I ended up making it a separate generator. What do you think @manthey of combining this? I like the idea of the environment override for the tiff header as an option, but I like my generate fix more. Can we combine these two approaches somehow?

Reply:
Sure -- I like your generator.  How would you like to combine our PRs?  Maybe I should remove my changes to the parser in this PR.

Reply:
If you want to remove the overlapping changes from your PR that might be easiest. It would also make it more obvious to users that there is the new env var handling when looking through PRs.

Reply:
@djhoese I've removed my handling of the line continuation parameters.